rest-helix/src/main/java/com/github/twitch4j/helix/domain/DropsEntitlementList.java

```diff
@@ -0,0 +1,44 @@
+package com.github.twitch4j.helix.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Data
+@Setter(AccessLevel.PRIVATE)
+@NoArgsConstructor
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DropsEntitlementList {
+
+    /**
+     * The entitlements
+     */
+    private List<DropsEntitlement> data;
+
+    /**
+     * If provided, is the key used to fetch the next page of data.
+     * If not provided, the current response is the last page of data available.
+     */
+    private Object pagination;
```

Comment:
Lets add `@Getter(AccessLevel.PRIVATE)` here as we have the method below to access the pagination curuser.